gradle/shipkit.gradle

```diff
@@ -0,0 +1,38 @@
+shipkit {
+   gitHub.repository = "linkedin/play-parseq"
+   //TODO generate and use your own readonly token
+   gitHub.readOnlyAuthToken = "TODO" //for reading the issues in GitHub
```

Comment:
Just curious, what's the way to make CI work without exposing all the credentials?

Reply:
Exposing read-only toke exposes no risk. Everybody in the world has read-only access to Open Source assets of GitHub. If you are worried that we are exposing the read only token, we can hide it behind an env variable and configure shipkit this way:

```Gradle
shipkit {
  gitHub.readOnlyAuthToken = System.getenv("GH_READ_ONLY_TOKEN")
}
```